src/crossChain/AxelarProvider.ts

```diff
@@ -0,0 +1,328 @@
+import { ethers } from 'ethers'
+import { Squid } from '@0xsquid/sdk'
+import { SquidCallType, ChainType } from '@0xsquid/sdk/dist/types'
+import { Provider } from 'decentraland-connect/dist/types'
+import { ChainId } from '@dcl/schemas'
+import { ERC20 } from '../abis/ERC20'
+import { MarketplaceV2 } from '../abis/MarketplaceV2'
+import { ERC721 } from '../abis/ERC721'
+import { CollectionStore } from '../abis/CollectionStore'
+import {
+  BuyNFTCrossChainData,
+  FromAmountParams,
+  MintNFTCrossChainData,
+  RouteResponse,
+  CrossChainProvider
+} from './types'
+import { getContract } from 'contracts'
+import { ContractName } from 'types'
+import { Marketplace } from 'abis/Marketplace'
```

Comment:
would you mind moving this import above?

```suggestion
import { Marketplace } from '../abis/Marketplace'
```